### DISCUSSION TITLE
Consulta tests tpi2 - sistema de archivos

### DISCUSSION THREAD

**Discussion by JuanIgnacioPiroso** [dds-jv/foro#425] [TPI3 - Sistema de Archivos] [answered]
Buenas noches! Queria consultar si los tests que ya vienen en el repositorio se tienen que dejar como estan, o los puedo modificar en base a lo que programe.

Muchas gracias!

**Comment by flbulgarelli** [comment, marked as answer]
:wave: ¡Hola! La idea es que completes los test que dicen _completar_, de mínima, y que agregues cualquier otro test que te sea útil. Preferentemente no deberías hacer cambios a los tests que ya vienen, salvo para agregarles alguna aserción o corregir algún bug, si tuvieran. 

¡Saludos!
